Source/ConversationV1/Models/DialogNodeCollection.swift

```diff
@@ -27,7 +27,7 @@ public struct DialogNodeCollection {
     /**
      Initialize a `DialogNodeCollection` with member variables.
 
-     - parameter dialogNodes:
+     - parameter dialogNodes: 
```

Comment:
Whoops! I made a local change to avoid trailing whitespace when descriptions are missing. Looks like I haven't pushed that up to the generator yet!